scripts/requirements.txt

```diff
@@ -0,0 +1,2 @@
+bs4==4.12.3
```

Comment:
@gradedSystem I didn't find where bs4 was used in the scripts. Maybe it is a mistake?

Reply:
@anuveyatsu yes it is a mistake I will change that in 10-15 minutes

Reply:
FIXED @anuveyatsu